operator/pkg/helm/path.go

```diff
@@ -0,0 +1,25 @@
+//go:build !windows
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"path/filepath"
+)
+
+func pathJoin(elem ...string) string {
+	return filepath.Join(elem...)
```

Comment:
Shouldn't Go's Windows implementation of this function add the correct slashes?

Reply:
I guess with fs.FS it uses forward slash while the go windows implementation assumes a real fs with backwards